notebooks/demo3/demo3-create-tables.ipynb

```diff
@@ -0,0 +1,546 @@
+{
```

Comment:
<p>I guess we can remove snakify column code for this type of csv. The only conversion it seems to be doing here is from "source" to "src" which may not be required. IIUC this function was written to work on really long names found in csvs such as in demo1.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>good point! will remove that</p>